purchase_landed_cost/__manifest__.py

```diff
@@ -5,12 +5,12 @@
 
 {
     'name': 'Purchase landed costs - Alternative option',
-    'version': '10.0.1.0.0',
-    "author": "OdooMRP team,"
-              "AvanzOSC,"
-              "Tecnativa,"
-              "Joaquín Gutierrez,"
-              "Odoo Community Association (OCA)",
+    'version': '10.0.2.0.0',
+    "author": u"OdooMRP team,"
+              u"AvanzOSC,"
+              u"Tecnativa,"
+              u"Joaquín Gutierrez,"
+              u"Odoo Community Association (OCA)",
     'category': 'Purchase Management',
     'website': 'http://www.odoomrp.com',
```

Comment:
Change to https://github.com/OCA/purchase-workflow
```suggestion
    'website': 'https://github.com/OCA/purchase-workflow',
```
As said in https://github.com/OCA/maintainer-tools/blob/master/template/module/__manifest__.py#L11